config/celeryconfig.py

```diff
@@ -0,0 +1,7 @@
+broker_url = "mongodb://0.0.0.0:27017/jobs"
+result_backend = "mongodb://0.0.0.0:27017"
+mongodb_backend_settings = {
+    "database": "jobs-result",
+    "taskmeta_collection": "celery_tasks",
+}
+result_persistent = False
```

Comment:
Any reason to use the separate celeryconfig file vs included configs via INI? 
(I don't mind the approach, I'm just curious because it worked as is in my previous usage)

Reply:
I failed to pass the mongodb_backend_settings from the INI. Weaver doesn't use any result backend (it implements its own), just the broker.